src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -212,7 +237,6 @@
             notes = "Accepts a NOMIS offender nomsNumber in the format A9999AA")
     @ApiResponses(
             value = {
-                    @ApiResponse(code = 200, message = "OK", response = OffenderLatestRecall.class),
```

Comment:
unnecessary annotations - spring fox / swagger will generate the 200 here anyway